### PR TITLE
Add ruleset for codeblocks.org

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

File: src/chrome/content/rules/Codeblocks.org.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!--
+	This domain contains a wildcard DNS record.
+
+	Invalid certificate:
+		www.codeblocks.org
+
+	Mismatch:
+		forums.codeblocks.org
+		wiki.codeblocks.org
```
Comment (cschanaj) [inline src/chrome/content/rules/Codeblocks.org.xml]: these 2 work for me, would you please check again ?

Comment (Jochen-A-Fuerbacher) [reply src/chrome/content/rules/Codeblocks.org.xml]: Both subdomains show on https an other content than on http. On https both subdomains show the content of the frontpage as shown on https://codeblocks.org.

Comment (cschanaj) [reply src/chrome/content/rules/Codeblocks.org.xml]: I see, can you mark them as `Different content`?

**Comment by cschanaj** [comment]
Thanks!

**Pull request merged**